//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// acdeg_O_AP.h
//
// Code generation for function 'acdeg_O_AP'
//

#ifndef ACDEG_O_AP_H
#define ACDEG_O_AP_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void acdeg_O_AP(double P_init, double V_init, double A_init, double P_wayp,
                double A_wayp, double V_max, double J_max, double J_min,
                creal_T t[14]);

#endif
// End of code generation (acdeg_O_AP.h)
